mmv1/third_party/terraform/services/container/resource_container_cluster.go.tmpl

```diff
@@ -1856,9 +1856,23 @@ func ResourceContainerCluster() *schema.Resource {
 										Elem: &schema.Schema{Type: schema.TypeString},
 									},
 								},
-
 							},
-
+						},
+						"auto_ipam_config": {
+							Type:        schema.TypeList,
+							MaxItems:    1,
+							Optional:    true,
+							DiffSuppressFunc: suppressDiffAutoIpamConfig,
```

Comment:
I checked the logs that the API returns a default value when no value is provided for this field auto_ipam_config.

Can you try to add back `Computed: true` and remove the DiffSuppressFunc? 
The failed tests such as `TestAccContainerCluster_additional_ip_ranges_config_on_create` should pass with `Computed: true`

Thanks.